src/main/window.js

```diff
@@ -132,7 +132,7 @@ class AppWindow {
         const textDirection = getDefaultTextDirection()
         win.webContents.send('AGANI::open-blank-window', {
           lineEnding,
-          markdown
+          markdown: ''
```

Comment:
Why need add the default value `''` even if it has a default value in the params.

>  createWindow (pathname, markdown = '', options = {}) {